rpc-rs/Cargo.toml

```diff
@@ -14,47 +14,47 @@ edition = "2021"
 exclude = [ "build.rs" ]
 
 [features]
-default = [ "chunkify" ]
+default = [ "otel" ]
 BigInteger = [ "num-bigint" ]
 BigDecimal = [ "bigdecimal" ]
-# non-working feature - used for internal testing of async-rewrite branch of nats-io/nats.rs
-#async_rewrite = [ "nats-experimental" ]
-chunkify = [ "nats" ]
-otel = ["opentelemetry", "tracing-opentelemetry", "nats", "opentelemetry-otlp"]
+prometheus = [ "dep:prometheus" ]
+otel = ["opentelemetry", "tracing-opentelemetry", "opentelemetry-otlp"]
 
 [dependencies]
+anyhow = "1.0.57"
 async-trait = "0.1"
 base64 = "0.13"
+bytes = "1.1.0"
 cfg-if = "1.0"
-minicbor = { version = "0.13", features = ["std", "partial-skip-support"] }
-rmp-serde = { version = "0.15.4" }
+minicbor = { version = "0.17.1", features = ["std"] }
+rmp-serde = "1.1.0"
 serde_bytes = "0.11"
 serde_json = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 thiserror = "1.0"
 time = "0.3.7"
 tokio-timer = "0.2"
 toml = "0.5"
-tracing = { version = "0.1.34", features = ["log"] }
+tracing = "0.1.34"
```

Comment:
hmm. What do you think about requiring users to use tracing? There are other potential conflicts about initializing logging that would be nice to avoid.